DTS-STN/Alpha-Site
# Turn province selector to select

# Description

[Use CDS implementation of dropdown for province dropdown on sign-up form](https://trello.com/c/49xx8F9g/253-253-use-cds-implementation-of-dropdown-for-province-dropdown-on-signup-form)
[Province dropdown in the sign-up form should clear like other form elements](https://trello.com/c/bPnc3jO7/252-252-province-dropdown-in-the-signup-form-should-clear-like-other-form-elements)

I followed the CDS implementation for a dropdown (https://design-system.service.gov.uk/components/select/) as well as the C-19 benefits finder (https://covid-benefits.alpha.canada.ca/en/province). I made sure the dropdown is a select and provinces are now listed alphabetically. I also made the dropdown clear with the rest of the form.
![image](https://user-images.githubusercontent.com/72703030/124609462-a191f280-de3d-11eb-863f-ae25a2a31a57.png)

## Acceptance Criteria

The dropdown should clear with the clear button, and the provinces should be listed alphabetically in french and english. There should be an "other" option. It should fit well on mobile devices as well.

## Test Instructions

1. Go to the sign-up form
2. Check the dropdown on different screen widths
3. Compare the dropdown with the C-19 benefits finder and the CDS implementation of a dropdown
4. Make sure it's alphabetical in french and english
5. It should also clear with the clear button

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
